watsong/static/images/dial-face.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" width="179.85" height="179.85" viewBox="0, 0, 179.85, 179.85">
+  <g id="Dial" transform="translate(-429.127, -269.176)">
+    <g>
+      <path d="M606.914,340.037 C617.442,388.562 586.64,436.434 538.116,446.962 C489.591,457.491 441.719,426.689 431.19,378.165 C420.662,329.64 451.464,281.768 499.988,271.239 C548.513,260.711 596.385,291.512 606.914,340.037 z" fill="#686868"/>
```

Comment:
Just wondering, what does all this code mean

Reply:
And in the other files

Reply:
It's a [vectorized image file](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics).